tests/testthat/testImportCNA.GISTIC2.R

```diff
@@ -0,0 +1,21 @@
+#
+# test importCNA.GISTIC2.R
+#
```

Comment:
I recommend to follow the structure given in testDummy.R - test handling of erroneous parameters and corrupt input, then check whether sane input gives correct output.